puppet/zulip/manifests/postgres_common.pp

```diff
@@ -7,8 +7,6 @@
                         "ptop",
                         # Python modules used in our monitoring/worker threads
                         "python-gevent",
-                        "python-tz",
-                        "python-dateutil",
```

Comment:
How do I make sure that dedicated postgres hosts use a virtualenv?

To test this I'll need to set up a dedicated postgres host, right?